includes/models/model.llms.course.php

```diff
@@ -598,6 +598,40 @@ public function has_capacity() {
 
 	}
 
+	/**
+	 * Returns true if the enrollment end date has passed, the course does not have an enrollment period,
+	 * or the course does not have an enrollment end date.
+	 *
+	 * @since [version]
+	 *
+	 * @return bool
+	 */
+	public function has_enrollment_period_ended() {
```

Comment:
The class changelog? I thought that new class methods were **not** mentioned in the class changelog because it is redundant with the new method's changelog.

Reply:
No, this kind of change should *not* be recorded.

The published changelogs are meant for users, primarily. We *can* add things like "introduced new filter" or function / method. But really that clutters up the changelog with information users don't care about.

A good changelog describes *what new features were added* and what *bugs were fixed*